src/monitord/sendcustomemail.c

```diff
@@ -0,0 +1,314 @@
+/* Copyright (C) 2009 Trend Micro Inc.
+ * All rights reserved.
+ *
+ * This program is a free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation
+ */
+
+/* Basic e-mailing operations */
+
+#include "shared.h"
+#include "os_net/os_net.h"
+
+/* Return codes (from SMTP server) */
+#define VALIDBANNER     "220"
+#define VALIDMAIL       "250"
+#define VALIDDATA       "354"
+
+/* Default values used to connect */
+#define SMTP_DEFAULT_PORT   "25"
+#define HELOMSG             "Helo notify.ossec.net\r\n"
```

Comment:
hi,

no idea what you trying to do here, but I would like to bring your attention on that hardcoded `ossec.net` value. It might lead to

0. data leak from ossec users to notify.ossec.net
0. dos on notify.ossec.net

If this is the expected behaviour, maybe add a comment describing the goal of this file ?

Reply:
That was unrelated to the intent of this pull request. I've closed this one to cut down on the confusion.